cmd/podman/parse.go

```diff
@@ -210,7 +215,8 @@ func parseEnv(env map[string]string, line string) error {
 		// if only a pass-through variable is given, clean it up.
 		val, exists := os.LookupEnv(name)
 		if !exists {
-			return errors.Errorf("environment variable %q does not exist", name)
+			// Docker compatibility
+			env[name] = ""
```

Comment:
it's pre-tea time for me, but should this be `val = ""`.  Otherwise you're just overwriting this variable at line 221.

Reply:
You're right, thanks! I guess that was during my pre-coffee time.